sdk/eventhub/azure-eventhubs/tests/asynctests/test_receive_async.py

```diff
@@ -309,14 +305,15 @@ def batched():
     for i in range(20):
         event_list.append(EventData("Event Number {}".format(i)))
 
-    async with receiver:
```

Comment:
Why was this removed?
If we don't have the `async with` then if an error occurs during the test then the receiver will not be closed. Which can cause issues in subsequent tests.

Reply:
Right. Changed back to use "async with".